src/reviews/reviews.module.ts

```diff
@@ -4,35 +4,36 @@ import {
   NestModule,
   RequestMethod,
 } from '@nestjs/common';
-import {ConfigModule} from '@nestjs/config';
-import {AuthenticationMiddleware} from 'src/common/middlewares/authentication.middleware';
+import { ConfigModule } from '@nestjs/config';
+import { JwtMiddleware } from 'src/common/middlewares/jwt.middleware';
 import reviewsConfig from './config/reviews.config';
-import {ReviewsController} from './reviews.controller';
-import {ReviewsService} from './reviews.service';
-import {PrismaModule} from 'src/prisma/prisma.module';
+import { ReviewsController } from './reviews.controller';
+import { ReviewsService } from './reviews.service';
+import { PrismaModule } from 'src/prisma/prisma.module';
 import prismaConfig from 'src/prisma/prisma.config';
-import {KafkaModule} from 'src/kafka/kafka.module';
+import { KafkaModule } from 'src/kafka/kafka.module';
+import { JwtModule, JwtService } from '@nestjs/jwt';
 
 @Module({
   imports: [
     ConfigModule.forFeature(reviewsConfig),
     PrismaModule,
     KafkaModule,
     ConfigModule.forFeature(prismaConfig),
+    JwtModule,
   ],
   controllers: [ReviewsController],
-  providers: [ReviewsService],
+  providers: [ReviewsService, JwtService],
 })
 export class ReviewsModule implements NestModule {
   configure(consumer: MiddlewareConsumer): MiddlewareConsumer | void {
     consumer
-      .apply(AuthenticationMiddleware)
+      .apply(JwtMiddleware)
       .forRoutes(
-        {method: RequestMethod.GET, path: '/reviews'},
-        {method: RequestMethod.GET, path: '/reviews/:id'},
-        {method: RequestMethod.POST, path: 'reviews'},
-        {method: RequestMethod.PATCH, path: '/reviews/:id'},
-        {method: RequestMethod.DELETE, path: '/reviews/:id'},
+        { method: RequestMethod.GET, path: '/reviews/:id' },
+        { method: RequestMethod.POST, path: 'reviews' },
+        { method: RequestMethod.PATCH, path: '/reviews/:id' },
+        { method: RequestMethod.DELETE, path: '/reviews/:id' },
```

Comment:
Maybe we could convert this middleware into a guard, and use that guard in individual controllers or the whole controller class. This way we could just use the guard and don't have to remember to add the new route to the configure method.